Update requests to include CSRF Token header

2Cubed opened the pull request:
Comply with API changes.

2Cubed committed:
Update requests to include CSRF Token header

Comply with API changes.

MCPH-Jenkins commented:
@2Cubed, thanks for your PR! By analyzing the blame information on this pull request, we identified @connor4312 to be a potential reviewer

2Cubed commented:
cc: @ProbablePrime

ProbablePrime commented on README.md:
````diff
@@ -8,14 +8,10 @@ We've built a chat bot that logs incoming messages to the console, and sends "Hi
-# start the example chatbot
-python example.py
-```
+Public domain: MIT License ([text](https://opensource.org/licenses/MIT))
````
:fire: this line.

2Cubed replied:
What even... I didn't add that... o.O
Fixing now.

ProbablePrime commented:
LGTM. we could add 461 retrying but as this library just connects to chat that shouldn't occur.

We'll look at it when we expand the client to immplement more endpoints

2Cubed committed:
Remove ghost README line

2Cubed commented:
@ProbablePrime Sounds good. I also plan to rewrite the entire thing, replacing `tornado` with `asyncio` (as per @connor4312's request).

2Cubed commented:
`flake8` is salty. ._.
One sec.

2Cubed committed:
Make flake8 happy

Newline at the end of the file? Really flake8? Would you like a pretzel with that salt?

2Cubed commented:
There we go. Ready for merge. @ProbablePrime

connor4312 commented:
As Richard mentioned retrying is ideal, but the scope of this client is so narrow that it makes no difference here. Lgmt.
